x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateTimeProcessorTests.java

```diff
@@ -34,17 +45,21 @@ protected DateTimeProcessor createTestInstance() {
     @Override
     protected DateTimeProcessor mutateInstance(DateTimeProcessor instance) throws IOException {
         DateTimeExtractor replaced = randomValueOtherThan(instance.extractor(), () -> randomFrom(DateTimeExtractor.values()));
-        return new DateTimeProcessor(replaced, UTC);
+        return processor(DateTime.now(), replaced);
     }
 
-    public void testApply() {
```

Comment:
Extracted these unit tests for different sql functions into separate test classes. Putting thigns in proper place.